### ISSUE TITLE
Exception: java.io.FileNotFoundException: build/serve/index.html (No such file or directory)

### ISSUE THREAD

**Issue by galuszkak** [avisi-cloud/structurizr-site-generatr#159]
Hey @dirkgroot @jp7677 ,

PR #154 introduce bug:
```
Mar 01, 2023 1:03:05 PM com.structurizr.dsl.AbstractViewParser generateViewKey
WARNING: Container-006 is an automatically generated view key - you will likely lose manual layout information when using automatically generated view keys.
Generating idex page...
java.io.FileNotFoundException: build/serve/index.html (No such file or directory)
        at java.base/java.io.FileOutputStream.open0(Native Method)
        at java.base/java.io.FileOutputStream.open(Unknown Source)
        at java.base/java.io.FileOutputStream.<init>(Unknown Source)
        at java.base/java.io.FileOutputStream.<init>(Unknown Source)
        at kotlin.io.FilesKt__FileReadWriteKt.writeBytes(FileReadWrite.kt:108)
        at kotlin.io.FilesKt__FileReadWriteKt.writeText(FileReadWrite.kt:134)
        at kotlin.io.FilesKt__FileReadWriteKt.writeText$default(FileReadWrite.kt:134)
        at nl.avisi.structurizr.site.generatr.site.SiteGeneratorKt.generateRedirectingIndexPage(SiteGenerator.kt:29)
        at nl.avisi.structurizr.site.generatr.ServeCommand.updateSite(ServeCommand.kt:71)
        at nl.avisi.structurizr.site.generatr.ServeCommand.execute(ServeCommand.kt:45)
        at kotlinx.cli.ArgParser.parse(ArgParser.kt:657)
        at kotlinx.cli.ArgParser.parse(ArgParser.kt:530)
        at nl.avisi.structurizr.site.generatr.AppKt.main(App.kt:13)
Starting server...
[main] INFO org.eclipse.jetty.server.Server - jetty-11.0.14; built: 2023-02-22T23:41:48.575Z; git: 4601fe8dd805ce75b69c64466c115a162586641b; jvm 19.0.2+7
[main] INFO org.eclipse.jetty.server.handler.ContextHandler - Started o.e.j.s.ServletContextHandler@3f36b447{/,null,AVAILABLE}
[main] INFO org.eclipse.jetty.server.AbstractConnector - Started ServerConnector@44ea608c{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
[main] INFO org.eclipse.jetty.server.Server - Started Server@435ce306{STARTING}[11.0.14,sto=0] @1638ms
Server started
Open http://localhost:8080 in your browser to view the site
```

And nothing is being generated.

**Comment by jp7677** [comment]
ah, the `serve` directory doesn't get created anymore :(, I guess that was generated implicitly when generating the diagrams.
In the mean time, until we have fixed this. you just have to create your `serve` directory manually.

**Comment by galuszkak** [comment]
Thanks for looking at it so quickly @jp7677 !

**Comment by dirkgroot** [comment]
I'm closing this issue, as it has been fixed by #160. Thanks for reporting, @galuszkak!
